master/pkg/model/experiment.go

```diff
@@ -786,9 +786,6 @@ func ExitedReasonToProto(r ExitedReason) experimentv1.TrialExitedEarly_ExitedRea
 
 	case UserCanceled:
 		return *experimentv1.TrialExitedEarly_EXITED_REASON_USER_REQUESTED_STOP.Enum()
-
-	case InitInvalidHP:
-		return *experimentv1.TrialExitedEarly_EXITED_REASON_INIT_INVALID_HP.Enum()
```

Comment:
Eyeballing the code, it seems that this `case` will no longer occur and therefore we should never land in the `default` clause with `panic`. We can probably leave it like this for now. Alternatively, we could also map `case InitInvalidHP` to `*experimentv1.TrialExitedEarly_EXITED_REASON_INVALID_HP.Enum()`

Reply:
Yup this case won't occur in this function for sure. Because this is only for custom-searcher.